README.md

```diff
@@ -37,8 +37,6 @@ For more information on the importance of a professional README for your reposit
 
 </div>
 
-<!-- TABLE OF CONTENTS -->
-
 # 📗 Table of Contents
```

Comment:
- [ ] Kindly remove the Microverse logo and replace the `Microverse  README template ` with your project name.